modules/deploying-models-using-multiple-gpu-nodes.adoc

```diff
@@ -92,49 +92,78 @@ spec:
   restartPolicy: Never
   initContainers:
     - name: fix-volume-permissions
-      image: quay.io/quay/busybox@sha256:xxxxx
+      image: quay.io/quay/busybox@sha256:92f3298bf80a1ba949140d77987f5de081f010337880cd771f7e7fc928f8c74d 
       command: ["sh"]
-      args: ["-c", "mkdir -p /mnt/models/granite-8b-code-base && chmod -R 777 /mnt/models"]
+      args: ["-c", "mkdir -p /mnt/models/$(MODEL_PATH) && chmod -R 777 /mnt/models"] <1>
       volumeMounts:
         - mountPath: "/mnt/models/"
           name: model-volume
+      env:
+        - name: MODEL_PATH
+          value: <model path> <2>
   containers:
     - resources:
         requests:
           memory: 40Gi
       name: download-model
       imagePullPolicy: IfNotPresent
-      image: quay.io/modh/kserve-storage-initializer@sha256:xxxxx
+      image: quay.io/opendatahub/kserve-storage-initializer:v0.14 <3>
       args:
-        - 's3://$<bucket_name>/granite-8b-code-base/'
-        - /mnt/models/granite-8b-code-base
+        - 's3://$(BUCKET_NAME)/$(MODEL_PATH)/' 
+        - /mnt/models/$(MODEL_PATH)
       env:
         - name: AWS_ACCESS_KEY_ID
-          value: <id>
+          value: <id> <4>
         - name: AWS_SECRET_ACCESS_KEY
-          value: <secret>
+          value: <secret> <5>
         - name: BUCKET_NAME
-          value: <bucket_name>
+          value: <bucket_name> <6>
+        - name: MODEL_PATH
+          value: <model path> <2>
         - name: S3_USE_HTTPS
           value: "1"
         - name: AWS_ENDPOINT_URL
-          value: <AWS endpoint>
+          value: <AWS endpoint> <7>
         - name: awsAnonymousCredential
           value: 'false'
         - name: AWS_DEFAULT_REGION
-          value: <region>
+          value: <region> <8>
+        - name: S3_VERIFY_SSL
+          value: 'true' <9>
       volumeMounts:
         - mountPath: "/mnt/models/"
           name: model-volume
 ----
+<1> The `chmod` operation is permitted only if your pod is running as root. Remove`chmod -R 777` from the arguments if you are not running the pod as root.
+<2> Specify the path to the model.
+ifndef::upstream[]
+<3> The value for `containers.image`, located in your `InferenceService`. To access this value, run the following command: `oc get configmap inferenceservice-config -n redhat-ods-operator -oyaml | grep kserve-storage-initializer:`
+endif::[]
+ifdef::upstream[]
+<3> The value for `containers.image`, located in your `donwload-model` container. To access this value, run the following command: `oc get configmap inferenceservice-config -n opendatahub -oyaml | grep kserve-storage-initializer:`
+endif::[]
+<4> The access key ID to your S3 bucket.
+<5> The secret access key to your S3 bucket.
+<6> The name of your S3 bucket.
+<7> The endpoint to your S3 bucket.
+<8> The region for your S3 bucket.
```

Comment:
The region option may confuse anyone that is not using an AWS S3 instance (which is most of our customers).

Maybe something like this:

> <8> The region for your S3 bucket if using an AWS S3 buckets.  If using other S3 compatible storage such as ODF or Minio, you can remove the AWS_DEFAULT_REGION environment variable.